# 最小有效距离（米）
min_range: 0.05

# 最大有效距离（米）
max_range: 30.0

# 有效角度范围（弧度）
# 格式：[开始角度1, 结束角度1, 开始角度2, 结束角度2, ...]
# 这里设置为 [-115°, -115°]
angle_ranges: [-2.0071286, 2.0071286]